tgui/packages/tgui/styles/interfaces/Phone.scss

```diff
@@ -0,0 +1,5 @@
+.phone__button > div {
```

Comment:
Changing the inner workings of a button here seems a little odd. Is there a reason to block overflow? If so, do these get tooltips so users can see the full text by hovering over it?

Reply:
primarily because allowing overflow would look ugly. ellipses work pretty well if the length of the name of the phone exceeds the width of the button.
![image](https://user-images.githubusercontent.com/66253095/215183896-2a948288-f149-4476-828f-4028822b60f2.png)
no tooltip tho.

Reply:
interface at default resolution should be big enough where nearly all phone names are gonna fit anyway, adding tooltips would be annoying from a ux perspective and a nightmare if someone makes a phone with an abhorrently long name